src/components/DpTooltip/utils/tooltip.js

```diff
@@ -83,8 +74,6 @@ const initTooltip = (el, value, options) => {
 const showTooltip = async (id, wrapperEl, { place = 'top', container = 'body', classes = '' }, zIndex)  => {
   if (!document.getElementById(wrapperEl.getAttribute('aria-describedby'))) {
```

Comment:
This function could be merged into `createTooltip`. This way, the wording would also be more appropriate (as we're not showing anything, just recreating the elements).

Reply:
like so?
843c1dee22976879111309c39c5ee7e470acbb14